vshard/replicaset.lua

```diff
@@ -485,7 +485,7 @@ local function can_backoff_after_error(e, func)
         end
     end
     if e.type == 'ShardingError' then
-        return e.code == vshard.error.code.STORAGE_IS_DISABLED
+        return e.code == lerror.code.STORAGE_IS_DISABLED
```

Comment:
I suggest to add a test case for this error

Reply:
Yes, I agree with Oleg. If you still want to continue with the PR, you would need to add a test. Preferably in `router2.test.lua`. In the same place where the other tests are located for catching the 'disabled' error.